### PR TITLE
rewrite to bash

### PR DESCRIPTION
**Pull request by domenkozar**
Inspired by @nmattia :)

### CODE REVIEW

File: lib/install-nix.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  sudo sh -c 'echo http2 = false >> /etc/nix/nix.conf'
+  # Set jobs to number of cores
+  sudo sh -c 'echo max-jobs = auto >> /etc/nix/nix.conf'
+  # Allow binary caches for runner user
+  sudo sh -c 'echo trusted-users = root runner >> /etc/nix/nix.conf'
+}
+
+if [[ $OSTYPE =~ darwin ]]; then
+  # Catalina workaround https://github.com/NixOS/nix/issues/2925
+  ./create-darwin-volume.sh
```
Comment (nmattia) [inline lib/install-nix.sh]: note sure where's it'll be run from but you may want to use `BASH_SOURCE`

**Comment by domenkozar** [comment]
> just... get rid of TypeScript stuck_out_tongue

I'd be happy to if I can get `nc` to work on macos and detect connectable unix socket :)

**Comment by nmattia** [comment]
Well you can do that from `.js`, right? the typescript stuff will still require some node modules

**Comment by domenkozar** [comment]
> the typescript stuff will still require some node modules

but only during development, not for using the action, right?

**Pull request merged**